database/connector.py

```diff
@@ -1,34 +1,181 @@
+                       "FOREIGN KEY (event_id) REFERENCES events(id), UNIQUE(name, event_id) ON CONFLICT REPLACE);")
+        cursor.execute("CREATE TABLE IF NOT EXISTS events "
+                       "(id INTEGER PRIMARY KEY AUTOINCREMENT, name VARCHAR);")
+        cursor.execute("CREATE TABLE IF NOT EXISTS expenses "
+                       "(id INTEGER PRIMARY KEY AUTOINCREMENT, description VARCHAR, payer VARCHAR, sum INTEGER, "
+                       "FOREIGN KEY (payer) REFERENCES users(id));")
+        cursor.execute("CREATE TABLE IF NOT EXISTS debtors "
+                       "(id INTEGER PRIMARY KEY AUTOINCREMENT, expense_id INTEGER, name INTEGER, sum INTEGER, "
+                       "FOREIGN KEY (expense_id) REFERENCES expenses(id), FOREIGN KEY (name) REFERENCES users(id));")
+        cursor.execute("CREATE TABLE IF NOT EXISTS payments "
+                       "(id INTEGER PRIMARY KEY AUTOINCREMENT, sender INTEGER, recipient INTEGER, sum INTEGER,"
+                       "FOREIGN KEY (sender) REFERENCES users(id), FOREIGN KEY (recipient) REFERENCES users(id));")
+
+    def save_event_info(
+            self,
+            event_name: str):
+        cursor = self.conn.cursor()
+        cursor.execute("INSERT INTO events (name) VALUES(?);", (event_name,))
+        self.conn.commit()
+        return cursor.lastrowid
+
+    def get_event_info(
+        self,
+        event_id: int
+    ) -> dict:
+        cursor = self.conn.cursor()
+        event = cursor.execute("SELECT * FROM events WHERE id = ?;", (event_id,))
+        return event.fetchone()
+
+    def save_user_info(
+            self,
+            user_name: str,
+            event_id: str):
+        cursor = self.conn.cursor()
+        cursor.execute("INSERT INTO users (name, event_id) VALUES(?, ?);", (user_name, event_id))
+        self.conn.commit()
+        return cursor.lastrowid
 
     def get_user_info(
         self,
-        user_id: int,
+        user_id: int
     ) -> dict:
-        """
-        Returns main information about user
+        cursor = self.conn.cursor()
+        user = cursor.execute("SELECT * FROM users WHERE id = ?;", (user_id,))
+        return user.fetchone()
 
-        :param user_id: user id
-        :return: user info
-        """
-        raise NotImplementedError
+    def save_debtor_info(
+            self,
+            expense_id: str,
+            debtor_name: str,
+            sum_of_debt: int):
+        cursor = self.conn.cursor()
+        cursor.execute("INSERT INTO debtors (expense_id, name, sum) VALUES(?, ?, ?);", (expense_id, debtor_name, sum_of_debt))
+        self.conn.commit()
+        return cursor.lastrowid
+
+    def get_debtor_info(
+        self,
+        debt_id: int
+    ) -> dict:
+        cursor = self.conn.cursor()
+        debt = cursor.execute("SELECT * FROM debts WHERE id = ?;", (debt_id,))
+        return debt.fetchone()
+
+    def save_expense_info(
```

Comment:
You may use an additional method for "save" purposes, like:
```
def _run_save_query(self, query):
    cursor = self.conn.cursor()
    cursor.execute(query)
    self.conn.commit()
    return cursor.lastrowid
```

So that methods like `save_*_info` will be like:
```
def save_expense_info(...):
    query = 'SELECT * FROM ...'.format(...)
    return self._run_save_query(query)
```

Reply:
in save or get purposes?